README.md

```diff
@@ -19,7 +19,7 @@ Lessons learned from Android developers in [Futurice](http://www.futurice.com).
 #### [Also keep dimens.xml DRY, define generic constants](#dimensxml)
 #### [Do not make a deep hierarchy of ViewGroups](#deephierarchy)
 #### [Avoid client-side processing for WebViews, and beware of leaks](#webviews)
-#### [Use Robolectric for unit tests, Robotium for connected (UI) tests, and AssertJ-Android for easier assertions in your Android tests](#test-frameworks)
+#### [Use JUnit for unit tests, Espresso for connected (UI) tests, and AssertJ-Android for easier assertions in your Android tests](#test-frameworks)
```

Comment:
Robolectric is typically a unit testing framework. It isn't suited for integration testing because you are testing against Robolectric, not Android (and the different versions of Android), so the results are not typically meaningful. In the majority of cases, I would just skip using Robolectric all together.